static_src/actions/domain_actions.js

```diff
@@ -0,0 +1,24 @@
+
+/*
+ * Actions for domain entities. Any actions such as fetching, creating,
+ * etc should go here.
+ */
+
+import AppDispatcher from '../dispatcher.js';
+import { domainActionTypes } from '../constants';
+
+export default {
+  fetch(domainGuid) {
+    AppDispatcher.handleViewAction({
+      type: domainActionTypes.DOMAIN_FETCH,
+      domainGuid
+    });
+  },
+
+  receivedDomain(domain) {
+    AppDispatcher.handleServerAction({
+      type: domainActionTypes.DOMAIN_RECEIVED,
+      domain
+    });
+  }
+};
```

Comment:
Yea I see what you mean. I think I separated them because there is a just Routes and just Domains view in the community UI, so figured we'd need to separate them conceptually at some point.